.releasenotes.yml

```
@@ -0,0 +1,6 @@
+project_type: library
+public_api_command: "cargo public-api --omit blanket-impls,auto-trait-impls,auto-derived-impls"
+gemini_model: "gemini-2.5-pro"
+diff_exclude_paths:
+  - aeon_models/**
+  - sbml_models/**
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)